builtin/logical/database/path_roles.go

```diff
@@ -209,12 +209,7 @@ func (b *databaseBackend) pathRoleCreateUpdate() framework.OperationFunc {
 			}
 
 			if revocationStmtsRaw, ok := data.GetOk("revocation_statements"); ok {
-				revStmts := revocationStmtsRaw.([]string)
-				revStmts = strutil.RemoveEmpty(revStmts)
-				// Only overwrite if they provided real data
-				if len(revStmts) > 0 {
-					role.Statements.Revocation = revStmts
-				}
+				role.Statements.Revocation = revocationStmtsRaw.([]string)
```

Comment:
Should you still run remove empty on this value?

Reply:
It will still be run below.